scripts/scil_transform_surface.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to load and transform cortical surface (vtk or freesurfer),
+This script is using ANTs transform (affine.txt, warp.nii.gz).
+
+Best usage for with ANTs from T1 to b0:
+> ConvertTransformFile 3 output0GenericAffine.mat vtk_transfo.txt --hm
+> scil_transform_surface.py lh_white_lps.vtk affine.txt lh_white_b0.vtk\\
+    --ants_warp warp.nii.gz
+"""
+
+import argparse
+
+import nibabel as nib
+import numpy as np
+from scipy.ndimage import map_coordinates
+from trimeshpy.io import load_mesh_from_file
+import trimeshpy.vtk_util as vtk_u
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+EPILOG = """
+References:
+[1] St-Onge, E., Daducci, A., Girard, G. and Descoteaux, M. 2018.
+    Surface-enhanced tractography (SET). NeuroImage.
+"""
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('surface',
+                   help='Input surface (FreeSurfer or supported by VTK).')
+
+    p.add_argument('ants_affine',
+                   help='Affine transform from ANTs (.txt).')
+
+    p.add_argument('out_surface',
+                   help='Output surface (formats supported by VTK).')
+
+    p.add_argument('--ants_warp',
+                   help='Warp image from ANTs.')
+
+    add_overwrite_arg(p)
+    return p
+
+
+def main():
+    parser = _build_args_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.surface, args.ants_affine],
+                        args.ants_warp)
+    assert_outputs_exist(parser, args, args.out_surface)
+
+    # Load mesh
+    mesh = load_mesh_from_file(args.surface)
+
+    # Affine transformation
+    if args.ants_affine is not None:
```

Comment:
Can use only
`
if  not args.ants_affine
`

if None is always equal to False

Reply:
changed to "if args.ants_affine:"